src/core_plugins/timelion/server/fit_functions/nearest.js

```diff
@@ -28,7 +28,7 @@ export default function nearest(dataTuples, targetTuples) {
     let i = 0;
     while (i < dataTuples.length - 1 &&
       (Math.abs(dataTuples[i + 1][0] - time) < Math.abs(dataTuples[i][0] - time) ||
-       // TODO: Certain offset= args can cause buckets with duplicate times, eg, offset=-1M
+       // TODO: Certain offset= args can cause buckets with duplicate times, e.g., offset=-1M
```

Comment:
https://kibana-ci.elastic.co/job/elastic+kibana+pull-request+multijob-intake/5104/console
17:26:43   43:1  error  Line 43 exceeds the maximum line length of 140  max-len